vue-frontend/src/components/android-app-development/DevelopmentSection.vue

```diff
@@ -18,7 +15,78 @@
         growth and customer engagement.</span
       >
     </div>
-    <div class="swiper-content tw-mt-8">
+    <!-- Desktop UI -->
+    <div class="sticky-parent tw-hidden md:tw-block">
+      <div class="sticky tw-sticky tw-top-0 tw-max-h-screen tw-overflow-hidden">
+        <div class="tw-flex">
+          <div
+            v-for="(item, index) in items"
+            :key="index"
+            class="tw-cursor-pointer tw-mt-32 xll:tw-mt-[36rem] tw-min-w-[100vw] md:tw-h-[70vh] xl:tw-h-screen"
+          >
+            <div class="tw-relative !-tw-mr-[14.5rem] tw-mb-8 tw-mt-12">
+              <div
+                class="tw-border-b-2 tw-border-dotted tw-border-[#000000]/[0.87] tw-text-center"
+              ></div>
+            </div>
+            <div
+              class="tw-absolute tw-top-[15.5%] xl:tw-top-[13.4%] xll:tw-top-[32.4%] tw-ml-[45%] xll:tw-ml-[47%] tw-hidden tw-h-20 !tw-bg-white tw-text-center md:tw-block"
+            >
+              <span
+                class="tw-text-center tw-font-inter-bold tw-text-[3.75rem] tw-leading-[5.625rem] !tw-bg-white"
+              >
+                0{{ index + 1 }}</span
```

Comment:
The box is correctly centered, but these numbers do not indicate the correct center in all boxes
![Android-App-Development-Company-Canopas](https://github.com/canopas/canopas-website/assets/115449373/4f08d84a-16e7-4f0d-ae48-78732c119bbb)

Reply:
which width?

Reply:
i'm not saying about width

Reply:
it is happening for all

Reply:
i have fixed that number only not width @cp-mansi-d